DTS-STN/Alpha-Site
# Updated privacy page

# Description

[374 Privacy Policy Page EN/FR](https://trello.com/c/mty2DKLm/374-374-privacy-policy-page-en-fr)

I updated the privacy policy page to match the figma design. I also updated the menu bar as well.

## Acceptance Criteria

The policy page needs to look like the figma design. The menu bar must be updated as well.

## Test Instructions

1. Run the code and go on the policy page and compare it to the figma design
2. Make sure the menu and the it's links work

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
